src/static/bootstrap/scss/bootstrap/_variables.scss

```diff
@@ -43,7 +43,8 @@ $link-hover-decoration: underline !default;
 //
 //## Font, line-height, and color for body text, headings, and more.
 
-$font-family-sans-serif:  "Helvetica Neue", Helvetica, Arial, sans-serif !default;
+$font-path:        "../../../fonts" !default;
```

Comment:
This file is part of bootstrap, we should not change it. Instead, we can set the variables in our ``_variables.scss`` to override them, e.g.:

```
$font-family-sans-serif:  "Open Sans", "OpenSans", "Helvetica Neue", Helvetica, Arial, sans-serif !default;
$font-path: static("fonts");
```